lib/platform/bitbucket/index.ts

```diff
@@ -712,16 +717,57 @@ export async function updatePr({
     )
   ).body;
 
-  await bitbucketHttp.putJson(
-    `/2.0/repositories/${config.repository}/pullrequests/${prNo}`,
-    {
-      body: {
-        title,
-        description: sanitize(description),
-        reviewers: pr.reviewers,
-      },
+  try {
+    await bitbucketHttp.putJson(
+      `/2.0/repositories/${config.repository}/pullrequests/${prNo}`,
+      {
+        body: {
+          title,
+          description: sanitize(description),
+          reviewers: pr.reviewers,
+        },
+      }
+    );
+  } catch (err) /* istanbul ignore next */ {
```

Comment:
```suggestion
  } catch (err) {
```

This is important functionality, so can it be tested?

Reply:
Yeah, couldn't post my suggestion with app earlier today.

Reply:
Tests added.  @rarkins - would you mind kicking off `2 workflows awaiting approval` as I'm a first time contributor